engine/table/src/main/java/io/deephaven/engine/table/impl/partitioned/PartitionedTableProxyImpl.java

```diff
@@ -586,5 +586,10 @@ public PartitionedTable.Proxy dropColumns(String... columnNames) {
         return basicTransform(ct -> ct.dropColumns(columnNames));
     }
 
+    @Override
+    public PartitionedTable.Proxy meta() {
+        return basicTransform(Table::meta);
+    }
```

Comment:
This return type is a bit weird; that said, it's *consistent*. I'm not sure it's worth trying to re-work the return type in a generic way just to make this specific implementation "cleaner".